package.json

```diff
@@ -25,12 +25,11 @@
     "ember-cli-import-polyfill": "^0.2.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.0.11",
-    "ember-getowner-polyfill": "^1.1.0",
```

Comment:
Actually, since Ember 2.3, it isn't. But does this package ensure retro-compatibility ? I runned the tests (`ember test` am I right ?)  without experiencing any issues.

I can't spot where the `errored` state comes from on TravisCI. But I see that one of your scenario is testing against ember 1.12 isn't it ? This could be the origin.